### PR TITLE
Changes for upgrade event during power event

### LINKED ISSUE #50: should allow to upgrade controller and satellites in the middle of Power Event
We need to upgrade the controller and pull payment satellite while the Power Event is going on.

- Remove the power event from the admin list.

- Pause the controller

- deploy the new pull payment contract and set in controller

- run upgrade event

- deploy new modified power event to handle the remaining crowdsale

### PR DESCRIPTION
**Pull request by bkcninja**
closes #50
Made a new PowerEventReplacement contract for UpgradeEvent to work with during PowerEvent

- Hard coded initial reserve and supply values from the old power event contract.

### CODE REVIEW

**Review by troggy — APPROVED** [review]
LGTM

File: contracts/policies/PowerEventReplacement.sol [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  // #CHANGED THIS FUNCTION TO REFLECT OLD PowerEvent VARIABLES
+  function startCollection() isState(EventState.Waiting) {
+    // check time
+    require(now > startTime);
+    // assert(now < startTime.add(minDuration));
+    // read initial values
+    var contr = Controller(controllerAddr);
+    powerAddr = contr.powerAddr();
+    nutzAddr = contr.nutzAddr();
+    initialSupply = 2399896170149257466012; //initialSupply as per old Power Event
```
Comment (troggy) [inline contracts/policies/PowerEventReplacement.sol:23]: @bkcninja where these came from?

For current PowerEvent these are:
0 and 3000000000000000 respectively

Comment (troggy) [reply contracts/policies/PowerEventReplacement.sol:23]: I must be missing something

Comment (troggy) [reply contracts/policies/PowerEventReplacement.sol:23]: sorry, I've been looking into wrong PowerEvent. Ignore me

**Comment by bkcninja** [comment]
fixes #50

**Review by johba37 — APPROVED** [review]
Nice

**Pull request merged**